src/components/layout/main-layout.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+interface MainLayoutProps {
+    oversikt: object;
+    dialogpanel: object;
+}
+
+function MainLayout(props: MainLayoutProps) {
+
+    const Wrapper = styled.div`
+        width: 100%;
+        display: flex;
+        flex-direction: row;
+        align-items: stretch;
+        padding: 1%;
+        box-sizing: border-box;
+    `;
+
+    const boxShaddow = 'box-shadow: 0 0.2em 1.5em #999;';
+
+    const float = 'float: right;';
```

Comment:
Jeg vet ikke så mye om det, men har fått inntrykk av at man bør unngå å bruke float og heller bruke flex-properties. Hva tenker du? Whatever _floats_ your boat egentlig for min del ;)

Reply:
`css-grid` for the win.
Verken `float` eller `flex` er spesielt gode på page-layout, men tidligere har vi ikke hatt noen alternativer. Men nå har vi `css-grid` :D

Reply:
@nutgaard enig, men tror ikke grid er en mulighet så lenge vi må støtte ie11?